<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="SQA-CRS-021" name="rebill.testngRebillFast" tests="113" failures="78" timestamp="25 Nov 2020 19:44:22 GMT" time="4145.477" errors="6">
  <testcase name="testMethod1" time="25.509" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not Clicked All Stat Codes">
      <![CDATA[java.lang.AssertionError: Could Not Clicked All Stat Codes
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1122)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="0.260" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="46.105" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="34.464" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="8.714" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="45.497" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="44.900" classname="rebill.testngRebillFast">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at rebill.testngRebillFast.validateResults(testngRebillFast.java:1894)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1690)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="35.748" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.825" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.278" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="45.614" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="29.873" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.450" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="52.255" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.613" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="47.174" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="54.744" classname="rebill.testngRebillFast">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="47.609" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="51.154" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="36.308" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="29.999" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod4" time="0.000" classname="rebill.testngRebillFast">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod4() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod4() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod4 -->
  <testcase name="testMethod1" time="51.550" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="38.421" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod3" time="0.000" classname="rebill.testngRebillFast">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod3() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod3() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod3 -->
  <testcase name="testMethod1" time="56.678" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="88.601" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.349" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="46.572" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="51.661" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="49.973" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.407" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.416" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="51.547" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="45.988" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="19.072" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed on Entering Tracking Number">
      <![CDATA[java.lang.AssertionError: Failed on Entering Tracking Number
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1034)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.905" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="35.697" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.454" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="18.931" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed on Entering Tracking Number">
      <![CDATA[java.lang.AssertionError: Failed on Entering Tracking Number
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1034)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="35.056" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.413" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="37.803" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="41.350" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="36.165" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="51.084" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="28.858" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.113" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="24.884" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not Clicked All Stat Codes">
      <![CDATA[java.lang.AssertionError: Could Not Clicked All Stat Codes
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1122)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="49.941" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="36.847" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="33.179" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="47.497" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="51.367" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="50.720" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="34.149" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="50.387" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="45.736" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="31.160" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="45.234" classname="rebill.testngRebillFast">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at rebill.testngRebillFast.validateResults(testngRebillFast.java:1894)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1690)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="52.057" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="38.541" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="54.024" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="53.257" classname="rebill.testngRebillFast">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.950" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.424" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="45.762" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="30.500" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="0.223" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="8.050" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="31.997" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.037" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="53.189" classname="rebill.testngRebillFast">
    <skipped/>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="25.441" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not Clicked All Stat Codes">
      <![CDATA[java.lang.AssertionError: Could Not Clicked All Stat Codes
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1122)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.598" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="29.986" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="33.435" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.408" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="46.358" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="31.981" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="0.271" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod2" time="0.000" classname="rebill.testngRebillFast">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod2() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rebill.testngRebillFast.dataProviderMethod2() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod2 -->
  <testcase name="testMethod1" time="50.520" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="24.941" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not Clicked All Stat Codes">
      <![CDATA[java.lang.AssertionError: Could Not Clicked All Stat Codes
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1122)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.600" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="37.970" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.584" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="51.221" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.510" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.783" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.112" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="49.553" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="33.381" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="52.117" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="46.381" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="46.193" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="29.819" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="34.070" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="34.936" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="52.309" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Could Not go to phone detail screen">
      <![CDATA[java.lang.AssertionError: Could Not go to phone detail screen
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1444)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.658" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="51.358" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="31.025" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="29.577" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="39.581" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="46.646" classname="rebill.testngRebillFast"/>
  <testcase name="testMethod1" time="33.380" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="32.821" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="47.521" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Faled At Last Rebill Screen: Did not reach override errors or failed here">
      <![CDATA[java.lang.AssertionError: Faled At Last Rebill Screen: Did not reach override errors or failed here
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1748)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="44.813" classname="rebill.testngRebillFast">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at rebill.testngRebillFast.validateResults(testngRebillFast.java:1894)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1690)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.198" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="31.176" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="30.976" classname="rebill.testngRebillFast">
    <failure type="java.lang.AssertionError" message="Failed Selecting Contact Method and Clicking Continue">
      <![CDATA[java.lang.AssertionError: Failed Selecting Contact Method and Clicking Continue
	at org.testng.Assert.fail(Assert.java:93)
	at rebill.testngRebillFast.doRebill(testngRebillFast.java:1500)
	at rebill.testngRebillFast.testMethod1(testngRebillFast.java:634)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
</testsuite> <!-- rebill.testngRebillFast -->
